### PR TITLE
Настройка Liveness, Readiness and Startup проб

### PR DESCRIPTION
**Pull request by vitalyzhakov**

### CODE REVIEW

**Comment by vitalyzhakov** [comment]
@Arhell , привет.
Похоже, разобрался с подисью *cncf-cla*.

File: content/ru/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 1]
```diff
@@ -0,0 +1,343 @@
+---
+title: Настройка Liveness, Readiness and Startup проб
```
Comment (sftim) [inline content/ru/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:2]:
```suggestion
title: Настройка Liveness, Readiness И Startup проб
```

File: content/ru/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+HTTP liveness проба использует этот прокси.
+В версиях после v1.13, определение локальной HTTP прокси в переменной окружения не влияет на HTTP liveness пробу.
+
+## Определение TCP liveness пробы
+
+Третий тип liveness проб использует TCP сокет. С этой конфигурацией
+kubelet будет пытаться открыть сокет к вашему контейнеру на определённый порт.
+Если он сможет установить соединение, контейнер будет считаться здоровым, если нет, будет считаться заваленным.
+
+{{< codenew file="pods/probe/tcp-liveness-readiness.yaml" >}}
```
Comment (sftim) [inline content/ru/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:184]: I'd expect to see this file copied into the localization unless it's already present.

File: content/ru/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 3]
```diff
@@ -0,0 +1,343 @@
...
+```
+NAME            READY     STATUS    RESTARTS   AGE
+liveness-exec   1/1       Running   1          1m
+```
+
+## Определение liveness HTTP запроса
+
+Другой вид liveness пробы использует запрос HTTP GET. Ниже представлен файл конфигурации для Pod, который запускает контейнер, основанный на образе `k8s.gcr.io/liveness`.
+
+{{< codenew file="pods/probe/http-liveness.yaml" >}}
```
Comment (sftim) [inline content/ru/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:128]: I'd expect to see this file copied into the localization unless it's already present.

File: content/ru/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md (hunk 2 above)
Comment (vitalyzhakov) [reply content/ru/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:184]: Copy/paste source code? Why not)

File: content/ru/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md (hunk 1 above)
Comment (vitalyzhakov) [reply content/ru/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md]: Fixed, thank you!

**Comment by vitalyzhakov** [comment]
@sftim , hello!
I have changed code after your comments.
Can you approve this PR?

**Comment by sftim** [comment]
@vitalyzhakov I am not a reviewer for this localization - my native language is English and there's no other language in which I am truly fluent.

File: layouts/partials/templates/blocks.html [hunk 4]
```diff
@@ -14,8 +14,10 @@
 {{ $header := . | replaceRE "</?h2.*?>" "" | htmlUnescape | safeHTML }}
 <li><a href="#{{ $id }}">{{ $header }}</a></li>
 {{ end }}
+{{ if  $section }}
 {{ $.ctx.Scratch.Add "sections" $section }}
 {{ end }}
+{{ end }}
```
Comment (sftim) [inline layouts/partials/templates/blocks.html:20]: Can you explain this change @vitalyzhakov ? It seems possibly unrelated. Maybe it should go into a separate pull request.

Comment (vitalyzhakov) [reply layouts/partials/templates/blocks.html:20]: Yes.
Before changes I see error
```
6:23:21 PM: Error: Error building site: failed to render pages: render of "page" failed: execute of template failed: template: docs/single.html:3:3: executing "content" at <partial "docs/content-page" (dict "ctx" . "page" .)>: error calling partial: execute of template failed: template: partials/docs/content-page.html:12:3: executing "partials/docs/content-page.html" at <partial . $>: error calling partial: execute of template failed: template: partials/templates/task.html:11:3: executing "partials/templates/task.html" at <partial "templates/blocks" .>: error calling partial: "/opt/build/repo/layouts/partials/templates/blocks.html:17:4": execute of template failed: template: partials/templates/blocks.html:17:4: executing "partials/templates/blocks.html" at <$.ctx.Scratch.Add>: error calling Add: reflect: call of reflect.Value.Type on zero Value
```
After changes I see result, what helps me to debug mistakes with including code.

Do we really need separate MR for 2 lines of code?

**Comment by vitalyzhakov** [comment]
> @vitalyzhakov I am not a reviewer for this localization - my native language is English and there's no other language in which I am truly fluent.

@sftim , yesterday @Potapy4 approve this PR on russian, additional changes was made after your comments.

**Comment by Arhell** [comment]
@vitalyzhakov
Hi, when another commit is made, then the approve fly off, it is necessary that @Potapy4  approve changes again

**Comment by sftim** [comment]
@vitalyzhakov can you rebase this against _master_, squashing to 2 commits:
- the revision to `layouts/partials/templates/blocks.html`
- the Russian localization changes

although, to tell you the truth, it sounds like the fix to `layouts/partials/templates/blocks.html` is not specific to Russian and should be in its own PR.
